src/main/java/com/linecorp/armeria/client/Clients.java

```diff
@@ -40,6 +43,28 @@
     }
 
     /**
+     * Creates a new client that connects to the static server group specified by {@code serverGroup}
+     * using the default {@link RemoteInvokerFactory}.For example,
+     * <p>
+     * <pre>{@code
+     * StaticServerGroup serverGroup = new StaticServerGroup();
+     * serverGroup.addHost("127.0.0.1", 1234, 2).addHost("127.0.0.1", 2345, 4)
+     *     .strategy(StaticServerGroup.SelectStrategyType.ROUND_ROBIN).registry("myGroup");
+     *
+     * Clients.newClient("tbinary+http://myGroup/...", ...,"myGroup",...);
```

Comment:
I think a client has either a hostname (uri here) or a server group, but not both. Making an API that accepts a path and a server group, rather than a uri with an ignored hostname (in this example http://myGroup/foo) would be less confusing. You could still implement this factory with the artificial URL if that's simplest.